ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -295,4 +295,18 @@ HWSRSR:
   description: "A heating water distribution riser."
   is_abstract: true
   implements:
-  - RSR
\ No newline at end of file
+  - RSR
+
+ADY:
+  id: "15252673377812888960"
```

Comment:
Remove ID

Reply:
Is the id field compulsory? Can i remove ID or stop defining IDS?

ADY:
  description: "Air dryer."
  is_abstract: true
  implements:
  - EQUIPMENT